src/portfolios/portfolio/components/TaskOrder/TaskOrderDetails.vue

```diff
@@ -480,7 +501,24 @@ export default class TaskOrderDetails extends Vue {
     const hasFollowOn = this.clins.some((clin) =>
       clin.clin_number === expectedClin && clin.clin_status === Statuses.OptionExercised.value
     );
-    return hasFollowOn ? Statuses.ExpiringPopOK.value : Statuses.ExpiringPop.value;
+    return hasFollowOn 
+      ? Statuses.ExpiringPopOK
+      : Statuses.ExpiringPop
+  }
+
+  public getClinStatus(clin: ClinDTO){
+    if (this.isUpcomingTO) {
+      return Statuses.Upcoming
```

Comment:
We only want CLINs in the Base period to have Status "Upcoming period" -- option periods will still have "Option pending" status. You could either just hard-code "Upcoming period" or append " period" to `Statuses.Upcoming` or add a new status `Statuses.UpcomingPeriod` in the acquisition package store.
![image](https://github.com/dod-ccpo/atat-web-ui/assets/90333349/63d2f7e1-7a84-4827-8e26-d158ec607c70)